### DISCUSSION TITLE
Embedded Flash Pro 6

### DISCUSSION THREAD

**Discussion by kowalski100** [polarfire-soc/polarfire-soc-documentation#79] [Reference designs, FPGA and development kits] [answered]
Does the onboard Embedded Flash pro has **DEBUG** support on Icicle Production Kit or its only used as programmer?

Do I need external FlashPro-6 for debugging or onboard embedded FlashPro-6 does the job!

- Ijaz

**Comment by nitindeshpande** [comment, marked as answer]
@kowalski100
Embedded Flashpro supports Debug.

**Comment by kowalski100** [comment]
Thank you @nitindeshpande.
